### PR TITLE
Feature fix events page table UI

### PR DESCRIPTION
**Pull request by code-rohitr**

### CODE REVIEW

File: src/utils/constants.js [hunk 1]
```diff
@@ -62,5 +62,5 @@ export default {
     light: "#f5f5f5",
     white: "#ffffff",
   },
-  server: serverUrl,
+  server: servers.development,
```
Comment (r21meghashyam) [inline src/utils/constants.js:65]: Please change it back to severUrl

**Review by r21meghashyam — CHANGES_REQUESTED** [review]

File: src/utils/constants.js (hunk 1 above)
Comment (code-rohitr) [reply src/utils/constants.js]: done

File: package.json [hunk 2]
```diff
@@ -48,6 +48,7 @@
     "@tailwindcss/typography": "^0.5.8",
     "autoprefixer": "^10.4.13",
     "prettier": "^1.16.4",
+    "remark-mdx": "^2.0.0-next.9",
```
Comment (manipalutsav-svc) [inline package.json:51]: I don't see you using this anywhere.

**Review by manipalutsav-svc — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (code-rohitr) [reply package.json:51]: Had to install it to get rid of some error while configuring the project in my local.

**Pull request closed without merging**